### DISCUSSION TITLE
Any idea when 4.0.0 is going to be officially released?

### DISCUSSION THREAD

**Discussion by ra-dave** [rikyoz/bit7z#137] [Q&A] [answered]
I see 4.0.0 is still marked as "RC".  When is it going to go non-RC?

**Comment by rikyoz** [comment, marked as answer]
Hi!
I'm currently working on adding more unit tests as possible. 
The library is pretty stable, though. The `master` branch has just a few bug fixes concerning the RC release.
If everything goes well, I plan to release the final version by the end of June.
